src/wieldmesh.cpp

```diff
@@ -501,12 +501,6 @@ void getItemMesh(Client *client, const ItemStack &item, ItemMesh *result)
 	const ContentFeatures &f = ndef->get(def.name);
 	content_t id = ndef->getId(def.name);
 
-	if (!g_extrusion_mesh_cache) {
```

Comment:
Hmmm, sounds like a structure issue, then. I can surely add this patch, but if the client dies someday with this error, it will due to not addressing the class hierarchy.
In this instance CItemDefManager should not exist before the first instance of WieldMeshSceneNode

Reply:
In this case we can ensure that an error is thrown if the structure is violated independently on what arguments are given to the function.